src/libsyntax/ast.rs

```diff
@@ -2606,7 +2606,7 @@ pub type ForeignItem = Item<ForeignItemKind>;
 #[derive(Clone, RustcEncodable, RustcDecodable, Debug)]
 pub enum ForeignItemKind {
     /// A static item (`static FOO: u8`).
-    Static(P<Ty>, Mutability),
+    Static(P<Ty>, Mutability, Option<P<Expr>>),
```

Comment:
Also not necessary.

Reply:
This commit is generally in the wrong direction given https://github.com/rust-lang/rust/pull/69194#issuecomment-586637192.

Reply:
And further commits too.

Reply:
(I assume this is also resolved since it's related to the comment above.)